### PR TITLE
Additional line break in stdout

### PR DESCRIPTION
**Pull request by PrasanthChettri**
1. added extra line break at the end of stdout write
2. added a wrapper for the writer functions that handles the file handler.

image
instead of 

image

### CODE REVIEW

**Comment by Rohitrajak1807** [comment]
@PrasanthChettri did you test the changes?

**Comment by PrasanthChettri** [comment]
Yeah, there's not test cases written for it right ? I played around with it and tested it manually. The changes seems to work.

**Comment by Rohitrajak1807** [comment]
> Yeah, I did not find any test cases for the package. I played around with it and tested it manually. The changes seems to work.

@PrasanthChettri do one thing. There's a bug with the HTML table writer code. It has been fixed in the branch py-scripts. So merge ser_de.py from that branch into your changes and push.

**Comment by Rohitrajak1807** [comment]
@PrasanthChettri bhai did you test after merging?

**Comment by PrasanthChettri** [comment]
Yeah seems to be working

**Comment by Rohitrajak1807** [comment]
Ok then @avaish1409 I think it's alright

File: src/gitcomp/ser_de.py [hunk 1]
```diff
@@ -66,36 +66,41 @@ def write(self):
...
-    def __to_json(self, g: object):
-        file_handle = self.__get_file_handle()
+    def __writer_func(writer) :
+        def writer_wraps(self, g : object):
+            file_handle = self.__get_file_handle()
+            writer(self, g, file_handle)
+            if file_handle is stdout  :
+                file_handle.write('\n')
+            self.__close_file_handle(file_handle)
+        return writer_wraps
```
Comment (Rohitrajak1807) [inline src/gitcomp/ser_de.py:76]: @PrasanthChettri good refactor with the decorator

**Review by Rohitrajak1807 — APPROVED** [review]
I have tested the changes as well. The line break works and the HTML table is properly rendered. It looks all good to me

**Pull request merged**